base/sysinfo.jl

```diff
@@ -570,6 +572,32 @@ function isexecutable(path::String)
 end
 isexecutable(path::AbstractString) = isexecutable(String(path))
 
+"""
+    Sys.isreadable(path::String)
+
+Return `true` if the given `path` has readable permissions.
```

Comment:
How about this text:
```suggestion
Return `true` if the access permissions for the given `path` permitted reading by the current user.
Note that this permission may change before the user calls `open`,
so it is recommended to just call `open` alone and handle the error if that fails,
rather than calling `isreadable` first.

!!! compat "Julia 1.11"
    This function requires at least Julia 1.11.
    
See also [`ispath`](@ref).
```

(and same for `iswritable` and `isexecutable`)